components/Member/Profile/ContactInfo.js

```diff
@@ -16,7 +26,27 @@ const FlexFormRow = styled(FormRow)`
   }
 `;
 
-const ContactInfoForm = ({ getFieldProps, errors, touched, values }) => {
+const ContactInfoForm = ({
+  editMode,
+  errors,
+  getFieldProps,
+  touched,
+  values,
+}) => {
+  const [validProfileSlug, setValidProfileSlug] = useState(false);
+
+  const validProfileSlugIndicator = newProfileSlug => {
+    const { loading, error, data } = useQuery(GET_PROFILE_SLUG_VALID, {
```

Comment:
@csell5 this here is giving me the `Invalid hook call` error that I'm working on now